chef/cookbooks/cpe_screensaver/recipes/default.rb

```diff
@@ -11,9 +11,24 @@
 # LICENSE file in the root directory of this source tree. An additional grant
 # of patent rights can be found in the PATENTS file in the same directory.
 #
+module_name = ''
+
+prefix = node['cpe_profiles']['prefix']
+organization = node['organization'] ? node['organization'] : 'Facebook'
+screensaver_profile = {
+  'PayloadIdentifier' => "#{prefix}.screensaver",
```

Comment:
not runtime safe. This need to be in the ruby_block to ensure this gets assigned during runtime